src/main/java/frc/robot/subsystems/Targeting.java

```diff
@@ -0,0 +1,179 @@
+// RobotBuilder Version: 2.0
+//
+// This file was generated by RobotBuilder. It contains sections of
+// code that are automatically generated and assigned by robotbuilder.
+// These sections will be updated in the future when you export to
+// Java from RobotBuilder. Do not put any code or make any change in
+// the blocks indicating autogenerated code or it will be lost on an
+// update. Deleting the comments indicating the section will prevent
+// it from being updated in the future.
+
+package frc.robot.subsystems;
+import frc.robot.commands.*;
+import edu.wpi.first.wpilibj.livewindow.LiveWindow;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj.Timer;
+import edu.wpi.first.wpilibj.command.Subsystem;
+import com.ctre.phoenix.sensors.PigeonIMU;
+import edu.wpi.first.networktables.NetworkTable;
+import edu.wpi.first.networktables.NetworkTableEntry;
+import edu.wpi.first.networktables.NetworkTableInstance;
+import edu.wpi.first.wpilibj.controller.PIDController;
+
+// BEGIN AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=IMPORTS
+// END AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=IMPORTS
+
+/**
+ *
+ */
+public class Targeting extends Subsystem {
+
+    // BEGIN AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=CONSTANTS
+    // END AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=CONSTANTS
+
+    // BEGIN AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=DECLARATIONS
+    // END AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=DECLARATIONS
+    private NetworkTable limeData;
+    // t stands for target
+    private NetworkTableEntry tAcquired;
+    // x and y values of the target
+    private NetworkTableEntry tx;
+    private NetworkTableEntry ty;
+
+    private double vP = 0.5;
+    private double vI = 0.0;
+    private double vD = 4.0;
+
+    private double CONFIRMED_THRESHOLD = 1.0;
+    private double CONFIRMED_TIME = .25; // Amount of seconds before it considers a target confirmed
```

Comment:
Both must be private **final** double. Remember final means that it cannot be changed